cluster-provision/images/vm-image-builder/fedora-with-test-tooling/cloud-config

```diff
@@ -36,7 +36,10 @@ write_files:
           WantedBy=cloud-init.service
 runcmd:
   - sudo systemctl daemon-reload
-  - sudo dnf install -y kernel-modules-$(uname -r) qemu-guest-agent stress-ng dmidecode virt-what vm-dump-metrics
+  - sudo dnf install -y http://ccrma.stanford.edu/planetccrma/mirror/fedora/linux/planetccrma/16/i386/planetccrma-repo-1.1-3.fc16.ccrma.noarch.rpm
+  - sudo dnf install -y kernel-modules-$(uname -r) qemu-guest-agent stress-ng dmidecode virt-what vm-dump-metrics kernel-rt tuned-profiles-realtime
+  - sudo grubby --set-default=$(ls /boot/vmlinuz-*rt*)
```

Comment:
I am not sure if its going to actually take affect because it requires a reboot
Maybe you can workaround this by intervene during the script on the part it boots the VM with virt-install
and restart the VM manually.

Reply:
It does a shutdown at the end of the script. I tested this image locally and when it  boots up, it does with the real time kernel:
```
[fedora@vmi-fedora ~]$ uname -r
5.6.19-300.rt10.2.fc32.ccrma.x86_64+rt
```